cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -665,7 +666,8 @@ def kick_api_server(tls):
     tls_client.reset_certificate_write_flag('server')
 
 
-@when('kubernetes-master.components.started')
+@when_any('kubernetes-master.components.started', 'ceph-storage.configured')
```

Comment:
This change could cause `configure_cdk_addons` to run before `kubernetes-master.components.started` has been set. If that happens, `cdk-addons.apply` will fail because `kube-apiserver` isn't up yet, and it will have to wait until the next hook to try again.

I think it'd be worthwhile to leave this as `@when('kubernetes-master.components.started')` so we don't introduce a delay in applying addons.